Editor/ManifestGenerator.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.IO;
+using System.Text;
+using UnityEditor;
+using UnityEditor.Build;
+using UnityEditor.Build.Reporting;
+using UnityEngine;
+
+namespace Elevator89.BuildPresetter
+{
+    public class ManifestGenerator : IPostprocessBuildWithReport
+    {
+        public int callbackOrder { get { return 0; } }
+
+        public void OnPostprocessBuild(BuildReport report)
+        {
+            if (report.summary.platform == BuildTarget.WebGL)
+            {
+                string streamingAssetsPath = "Assets/StreamingAssets";
+                string outputPath = report.summary.outputPath + "/manifest.txt";
```

Comment:
The problem of accessing StreamingAssets exists not only for WebGL builds, but for Android builds as well.

For Android, `report.summary.outputPath` leads to an .apk file, not a directory. So it will not be possible to put any files there.

It would be better to have a unified solution for both platforms, if possible.

Reply:
исправила скрипт